File: solutions/0807-custom-sort-string/solution.py
class Solution(object):
    def customSortString(self, order, s):
        """
        :type order: str
        :type s: str
        :rtype: str
        """
        ls=""
        for i in order:
            if i in s:
                ls+=i*s.count(i)
                s=s.replace(i,"")
        return ls+s

File: solutions/0807-custom-sort-string/solution.java
class Solution {
    public String customSortString(String order, String s) {
        String ls="";
        for(char i:order.toCharArray())
        {
            if(s.contains(i+""))
            {
                ls+=(i+"").repeat(count(s,i));
                s=s.replace(i+"","");
            }
        }
        return ls+s;
    }
    int count(String s,char i)
    {
        int n=s.length();
        s=s.replace(i+"","");
        return n-s.length();
    }
}
